Framework/API/inc/MantidAPI/FileFinderHelpers.h

```diff
@@ -0,0 +1,41 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include "MantidAPI/DllConfig.h"
+
+#include <string>
+#include <vector>
+
+namespace Mantid::API {
+
+class MANTID_API_DLL FileFinderHelpers {
+public:
+  /// Search instrument directories for Parameter file,
+  /// return full path name if found, else "".
+  static std::string getIPFPath(const std::string &instName,
```

Comment:
Just a question on symmetry. We have `getInstrumentFilename` below but then an abbreviated `getIPFPath` so we've got both path and filename. If we did go with having Instrument in class name then how about `getParamterFilename()` for this?

Reply:
I've gone for `getParameterPath` as this returns a full path
`getInstrumentFilename` should be really called `getInstrumentDefPath` or something similar, but for backwards compatibility and as everyone knows it as the former I've left it as is